E3Next/Processors/NowCast.cs

```diff
@@ -95,6 +95,14 @@ private static void RegisterEvents()
 
         }
 
+        public static bool IsNowCastInQueue()
+        {
+            if(EventProcessor.CommandList["/nowcast"].queuedEvents.Count > 0)
```

Comment:
is this dict guaranteed to have this key?

Reply:
yes